### PR TITLE
Migrate Orleans.TelemetryConsumers.Counters to netstandard

### PR DESCRIPTION
**Pull request by benjaminpetit**
See #4835

### CODE REVIEW

File: src/TelemetryConsumers/Orleans.TelemetryConsumers.Counters/OrleansPerformanceCounterInstaller.cs [hunk 1]
```diff
@@ -1,94 +0,0 @@
...
-using Orleans.Metadata;
-using Orleans.Runtime.Configuration;
-
-namespace OrleansTelemetryConsumers.Counters
-{
-    /// <summary>
-    /// Providers installer hooks for registering Orleans custom performance counters.
-    /// </summary>
-    [RunInstaller(true)]
-    public class OrleansPerformanceCounterInstaller : Installer
```
Comment (xiazen) [inline src/TelemetryConsumers/Orleans.TelemetryConsumers.Counters/OrleansPerformanceCounterInstaller.cs]: why is this removed? This installer will be invoked if the assembly is installed in Visual Studio Custom Action Installer or the Installutil.exe, to my understanding.

Comment (benjaminpetit) [reply src/TelemetryConsumers/Orleans.TelemetryConsumers.Counters/OrleansPerformanceCounterInstaller.cs]: Because there is no official nuget for `System.Configuration.Install`, as far as I know.

User can still use `OrleansCounterControl` to install them using a .exe
